src/zino/utils.py

```diff
@@ -63,3 +65,12 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return actual_decorator
+
+
+def file_is_readable_by_others(file: str) -> bool:
+    """Returns a boolean value indicating if a file is readable by other users than its owner"""
+    st_mode = getattr(os.stat(path=file), "st_mode", None)
+    if not st_mode:
+        return False
```

Comment:
This branch isn't covered by tests. Not quite sure what this branch is here for?  Is there ever a case where `os.stat()` will return an object that doesn't have the `st_mode` attribute?

Reply:
I was uncertain about that, because I could not find any information about it in the documentation (https://docs.python.org/3/library/os.html#os.stat), but I just checked the source code and there it says that `st_mode` is always present in a result of `os.stat()`.